### ISSUE TITLE
Warnings about versioned JavaScript

### ISSUE THREAD

**Issue by 1ec5** [1ec5/avim#188]
Thunderbird 58.0b2 issues the following warnings when the extension loads:

```
Versioned JavaScripts are no longer supported. Please remove the version parameter.  avim.xul:6
Versioned JavaScripts are no longer supported. Please remove the version parameter.  avim.xul:7
```

**Comment by 1ec5** [comment]
The version parameters are present for compatibility with very old versions of Firefox and Thunderbird that don’t use JavaScript 1.7+ by default:

https://github.com/1ec5/avim/blob/33c717ee311a6002793442c2e88ce3849ce0e45e/content/avim.xul#L6-L7

**Comment by 1ec5** [comment]
Fixed in #192.
